CMakeLists.txt

```diff
@@ -96,7 +96,8 @@ set(CPACK_GENERATOR DEB)
 set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)
 set(CPACK_DEBIAN_PACKAGE_MAINTAINER "=nil; Foundation")
 
-set(CPACK_DEBIAN_PACKAGE_DEPENDENCIES_MULTITHREADED_VERSION "libhwloc-dev, libyaml-cpp-dev, libfmt-dev")
+set(CPACK_DEBIAN_PACKAGE_DEPENDENCIES_MULTITHREADED_VERSION "libhwloc-dev (=2.7.0-2ubuntu1), libyaml-cpp-dev (=0.7.0+dfsg-8build1), libfmt-dev(=6.1.2+ds-2), libc-dev, lksctp-tools, libatomic1, libprotobuf-dev (=3.12.4-1ubuntu7.22.04.1), liblz4-dev (=1.9.3-2build2), libgnutls28-dev (=3.7.3-4ubuntu1.3), libc-ares-dev (=1.18.1-1ubuntu0.22.04.2)")
```

Comment:
Do we need all the list or can we drop some dependencies? This is a really long list we have here.

Reply:
It's safer this way. In theory, all of these can be used by actor. We can remove things like libc-dev and libatomic1, because they will be on the system for sure, but it's safer to keep them.

Reply:
Alright, let's review it latter.